src/main/java/uk/gov/justice/hmpps/prison/repository/jpa/repository/OffenderBookingRepository.java

```diff
@@ -68,6 +70,13 @@ List<OffenderBookingId> findDistinctByActiveTrueAndOffenderNomsIdInAndSentences_
 
     Optional<OffenderBooking> findByBookingId(Long bookingId);
 
+    String lockWaitTimeMillis = "10000";
```

Comment:
Should we pick this up from application yaml (and use this value as the default?)

Reply:
I was thinking of setting 'jakarta.persistence.lock.timeout' in application.yaml , in which case it applies globally but a custom config value might be better (at least in the short to medium term) as it would only apply for the endpoints we want. Will add this in with the next lock subtask.